docs/diagrams/CommandTextHistoryActivityDiagram.puml

```diff
@@ -0,0 +1,28 @@
+@startuml
+
+skin rose
+skinparam ActivityFontSize 15
+skinparam ArrowFontSize 12
+
+start
+:User makes an action;
+switch()
+case ([press up button])
+    if () then ([currentCommandIndex < size])
+        :CommandTextHistory gets previous command;
+    else ([else])
+        :Nothing happens;
+    endif
+case ([press down button])
+    if () then ([currentCommandIndex >= 0])
+        :CommandTextHistory gets next command;
+    else ([else])
+        :Nothing happens;
+    endif
```

Comment:
Do you think we should add the case where `currentCommandIndex >= commandHistory.size()`? Since this case doesn't technically get the next command (as what it's currently suggesting) but just stays on the last command